src/library/dao/cuedao.cpp

```diff
@@ -197,21 +197,6 @@ bool CueDAO::saveCue(Cue* cue) const {
     return true;
 }
 
-bool CueDAO::deleteCue(Cue* cue) const {
-    //qDebug() << "CueDAO::deleteCue" << QThread::currentThread() << m_database.connectionName();
-    if (!cue->getId().isValid()) {
```

Comment:
I don't see a good reason to delete this honestly.

Reply:
The reason is that it is no longer used.

Reply:
But is there no valid usecase for it to be used? What is gained from removing the code?

Reply:
It improved compile time and the time of developers for reading and understanding the class.

Reply:
The improvements in compile time are utterly negligible. This will only cause confusions for developers when they see a class that can save a cue, but not delete it. The interface is incomplete and thus confusing.

Reply:
This was a private function. The intended use is to use of this class is to use it via the track with all cues at once.

Reply:
Fine, then lets hope nobody will need this in the future then.